todo_app/app.py

```diff
@@ -52,58 +59,43 @@ def get_card(id):
         return next((item for item in items if item['id'] == id), None)
 
     def save_card(item):
+        date = datetime.datetime.now()
         existing_items = get_cards()
         updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-
-        url = "https://api.trello.com/1/cards/"+item['id']
-        headers = {"Accept": "application/json" }
-        if (item['status'] == "Done"):
-            idList=done_status
-        elif (item['status'] == "Doing"):
-            idList=doing_status
-        else:
-            idList= toDo_status
-        querystring = {"key": API_KEY, "token": TOKEN, 'status': item['status'], "name": item['title'], "idList": idList}
-        response = requests.request("PUT",url,headers=headers,params=querystring)
+        todos.update_one({"_id":item['id']},{"$set":{"title":item['title'],"status":item['status'],'DateUpdated':date}})
         return item
 
     def add_card(title):
-        url = f"https://api.trello.com/1/cards"
-        querystring = {"name": title, "idList": toDo_status, "key": API_KEY, "token": TOKEN}
-        response = requests.request("POST", url, params=querystring)
-        card_id = response.json()["id"]
         date = datetime.datetime.now()
-        itemDict.append(Item(card_id, title, 'To Do', date.date()))
-        item = Item(card_id, title, 'To Do', date.date())
+        print (date)
+        id_val = str(len(_DEFAULT_ITEMS))
```

Comment:
Using the length of this variable means that if the program stops and restarts then we start our ids counting from scratch which will cause bugs!
The best solution might be to not track the id ourselves, and allow Mongo to decide that for us. That will involve tweaking some other parts of your code to handle the BSON Object though. A quicker fix might be to switch to use the count of the number of items in the database, but this would still hit issues if we ever allow items to be deleted from the DB!
`id_val = str(todos.count())`